student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(*person, classroom)
+    super(*person)
+    @classroom = classroom
+  end
```

Comment:
- [ ] the constructor should include age, classroom, name and parent_permission separately similar to the example below:
```
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```